Unify: allow imports of ApiModules hierarchy (#2840)
demisto/demisto-sdk: CHANGELOG.md

* Added **validate** check for the `modules` field in `pack_metadata.json` files.
* Changed **lint** to skip deprecated content, unless when using the `-i` flag.
* Refactored the logging framework. Demisto-SDK logs will now be written to `.demist_sdk_debug.log` under the content path (when detected) or the current directory.

> * Added support for inserting any ApiModule code imported by an ApiModule in the integration using the **unify** command.

Reviewer: ```suggestion
* Added support for API Modules imported in API modules in the **unify** command.
```
